### PR TITLE
Fix typos in the documentation.

### PR DESCRIPTION
**Pull request by mundry**
## Changes

- Typo fixes in the documentation to improve quality

## Docs

Typos fixed to improve the quality of the documentation.

### CODE REVIEW

File: docs/src/pages/blog/island-architecture.md [hunk 1]
```diff
@@ -166,13 +166,13 @@ This 'micro' architecture is similar to both 'micro-frontends' and 'micro-servic
 
 With Island-Architecture, he proposes a form of progressive enhancement for the dynamic components by using a technique known as _Partial Hydration_.
 
-Lets look at this following analogy:
+Let's look at this following analogy:
```
Comment (sno2) [inline docs/src/pages/blog/island-architecture.md:169]: I think "this" should also be "the".

Comment (FredKSchott) [reply docs/src/pages/blog/island-architecture.md:169]: agreed, will change as a part of merging

**Comment by FredKSchott** [comment]
This is amazing, thank you! Just curious, did you go through this all by hand or use some automated tool? If you used a tool, please share!

**Comment by mundry** [comment]
My pleasure!

I read the entire documentation. Before committing the changes I made manually, I ran the documentation through Webstorm's spellcheck, which is powered by LanguageTool.

**Pull request merged**